lib/galaxy/datatypes/binary.py

```diff
@@ -1907,6 +1907,59 @@ def sniff(self, filename):
             return False
 
 
+class Dcd(Binary):
+    """
+    Class describing a dcd file from the CHARMM molecular simulation program
+
+    >>> from galaxy.datatypes.sniff import get_test_fname
+    >>> fname = get_test_fname('test_glucose_vacuum.dcd')
+    >>> Dcd().sniff(fname)
+    True
+    >>> fname = get_test_fname('interval.interval')
+    >>> Dcd().sniff(fname)
+    False
+    """
+    file_ext = "dcd"
+    edam_format = "format_3842"  # molecular simulation data
+
+    def __init__(self, **kwd):
+        Binary.__init__(self, **kwd)
+        self._magic_number = b'CORD'
+
+    def sniff(self, filename):
+        # The second struct of any dcd file contain 'CORD'
+        # First struct is not sufficient, velocity files will match.
+        # Not checking for endianness
+        try:
+            intsize = 4
+            header = open(filename, 'rb')
```

Comment:
can we use `with open() as header:` here?

Reply:
yes, updating :)